docs/conf.py

```diff
@@ -61,9 +62,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '7.0'
+version = "7.0.1"
 # The full version, including alpha/beta/rc tags.
-release = 'v7.0.1'
+release = "7.0.1"
```

Comment:
I'd like to make this and the openstack version update automatically, like in the heat-plugins repo. Can we do that here?